doc/keycmds.dsv

```diff
@@ -5,6 +5,7 @@ reload||r||Reload the currently selected feed.
 reload-all||R||Reload all feeds.
 mark-feed-read||A||Mark all articles in the currently selected feed read.
 mark-all-feeds-read||C||Mark articles in all feeds read.
+mark-feed-read-until-article||a||Mark feed read until current article.
```

Comment:
"Until" didn't sit well with me, and after a short discussion on our IRC, I was told that Reader on iPhone uses "Mark all above as read" for the similar function. What do you think?

Another point: I would rather *not* bind this to any key by default, because this command looks pretty niche to me. Can you please update the docs and the code accordingly? (In docs, we use "n/a" in second column for commands that aren't bound to anything by default).

Reply:
1. It does not matter to me which name of the command to use.  "Mark all above as read" seems OK for me. I will use that. While I was trying to following to "tradition" used in the code and in docs: to make something close to command "Mark all articles in the currently selected feed read."

2. Well for someone it's a "niche" function. But for me it's a vital function. I cannot use newsboat without it. It is in my "workflow". While, for example, tagging is not needed for me. The tagging functions are niche for me. But I don't mind I may remove 'a' key from it. Because, anyway, it's possible to use bind-key command to do the personal bind.

Reply:
> I will use that. While I was trying to following to "tradition" used in the code and in docs

Yeah, I noticed that—and thank you for taking the time to look around the code and see what's already there! It's just that "until" as a word didn't seem appropriate: it's a bit vague and implied that current article won't be marked read.

> Well for someone it's a "niche" function. But for me it's a vital function.

Sorry, I didn't mean to say that this function is unimportant or something like that! I only observed that this particular function isn't widely used, otherwise I'd have seen a request for it already.